### PR TITLE
Updated build instructions in README

### PR DESCRIPTION
**Pull request by jaylogue**
This change addresses issue #11.

### CODE REVIEW

**Review by samhansen — APPROVED** [review]
Looks great, thanks!

File: README.md [hunk 1]
```diff
@@ -45,34 +45,49 @@ ___
...
 
 * Clone the demo application repository and fetch the referenced submodules.
 
-        git clone git@github.com:openweave/openweave-esp32-demo.git openweave-esp32-demo
-        cd openweave-esp32-demo
-        git submodule init
-        git submodule update
+        $ mkdir ${HOME}/projects
+        $ cd ${HOME}/projects
+        $ git clone https://github.com/openweave-esp32-demo.git
```
Comment (samhansen) [inline README.md]: The github link needs updating:

git clone https://github.com/openweave/openweave-esp32-demo.git

**Review by samhansen — APPROVED** [review]
I ran through this from fresh repos.  Approval stands (though one thing stuck out below)

File: README.md (hunk 1 above)
Comment (jaylogue) [reply README.md]: Ooops, looks like a cut-and-paste error.  Will fix.

**Pull request merged**